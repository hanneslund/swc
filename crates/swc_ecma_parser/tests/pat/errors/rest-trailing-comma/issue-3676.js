for ([...rest,] of a) {}
for ({...rest,} of b) {}
