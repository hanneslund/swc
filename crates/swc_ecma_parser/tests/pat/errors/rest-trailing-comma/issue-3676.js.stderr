
  x Trailing comma isn't permitted after a rest element
   ,-[$DIR/tests/pat/errors/rest-trailing-comma/issue-3676.js:1:1]
 1 | for ([...rest,] of a) {}
   :              ^
   `----

  x Trailing comma isn't permitted after a rest element
   ,-[$DIR/tests/pat/errors/rest-trailing-comma/issue-3676.js:2:1]
 2 | for ({...rest,} of b) {}
   :              ^
   `----
